### ISSUE TITLE
Request for ofxAvVideoWriter

### ISSUE THREAD

**Issue by corybarr** [kritzikratzi/ofxAvCodec#11]
I'm wondering if there's any code perhaps unpublished that creates a video file. I'd like to append a frame at various intervals to create a video. I've found a few examples to start with, but figure I'd ask first. I'm up for doing some work and sending a PR.

**Comment by kritzikratzi** [comment]
PRs are always welcome!

**Comment by corybarr** [comment]
I got this working, and can issue a pull request after a bit of cleanup. However, I'd really like to create a HAP-encoded file. I'm able to get the decoder, but not the encoder. Any ideas why?

Specifically, in this code "dec" is non-null while "enc" is null.

```
	AVCodecID codecID = AV_CODEC_ID_HAP;
	AVCodec *dec = avcodec_find_decoder(codecID);
	AVCodec *enc = avcodec_find_encoder(codecID);

```
I was able to concatenate two HAP-encoded files and make a new one that way. However, that doesn't require actual encoding, just streaming and concatenating.

I will admit to currently fairly limited knowledge of FFMPEG, so any ideas however newb might be helpful.

Thanks!
